### ISSUE TITLE
Support for newlines in VSTS PRs

### ISSUE THREAD

**Issue by tylerl0706** [Microsoft/vsts-cli#160]
I have been creating PRs using the VSTS CLI and love how simple it is. However, one of the biggest problems for me is that there's no way to include newlines in the `--description` flag.

I've tried:

```
vsts pr create -s foo -t bar -d "Hello <br/> world"
```
```
vsts pr create -s foo -t bar -d "Hello \n world"
```
```
vsts pr create -s foo -t bar -d "Hello `n world"
```

all of which show up as plaintext in the VSTS PR rather than actual new lines. Here's my (not great) workaround in VSTS:
![image](https://user-images.githubusercontent.com/2644648/40943615-a0156104-6806-11e8-9e93-3ad352b283c5.png)

then I host a markdown file somewhere which has a rich description.

Am I doing something wrong?

**Comment by tedchamb** [comment]
It is easy to add multiline values in the bash shell, where you can type the enter key within a quoted argument value, and the value will continue on the next line.

In a windows command prompt this doesn't work however.  I will need to investigate to see if there is a good way to address this.  One option is to make the description argument take a list of values, and each value would be a new line.  That would allow you to do something like:
`vsts pr create -s foo -t bar -d "line 1" "line 2" "line 3"`

**Comment by SteveL-MSFT** [comment]
Personally, I think using `\n` makes it easy

**Comment by tylerl0706** [comment]
Do you know if the bash solution should be working in PowerShell?

I'm creating the newlines in a PowerShell script today similar to what you describe in bash but that doesn't work.

**Comment by Koubek** [comment]
Hello please, I have the same problems with PowerShell. I am trying to send a multiline markdown but still no success. Please, could you properly explain how to pass multiline markdowns? 

Or is something that won't work from PowerShell? In this case, I would like to know whether we can expect any solution for that and eventually when.

Thank you very much.

**Comment by tylerl0706** [comment]
@Koubek since the VSTS cli uses a batch file as an entry point for the CLI, what you want is not possible right now, I'm afraid. That said, take a look at what [a couple comments up](https://github.com/Microsoft/vsts-cli/issues/160#issuecomment-394549972) 

I did to get it working by by passing the batch script and calling the Python portion (which is what the CLI is written in) directly

**Comment by Koubek** [comment]
@tylerl0706 Thank you very much. 

Actually, this is quite disappointing to have those limitations on VSTS. I can see VSTS as something strategic for all developers using MS platform. In the same position, I can see PowerShell as a primary scripting language. 

If VSTS CLI is the only way how to do VSTS integration I would expect at least a little bit better support for PowerShell.

Please, don't get me wrong. I am really grateful for this tool as it already let us do some automation. My head just doesn't get how is it possible to have ```--description``` parameter that let me pass a multiline markdown but cannot be able to do it from PowerShell.

**Comment by Koubek** [comment]
@tedchamb Thank you very much for your feedback!!! I can see your statement made on June 4th but I didn't understand there is any plan on the addressing of the issue.

Please, is there any estimation?

And again, thank you very much.

**Comment by tedchamb** [comment]
#205

**Comment by tylerl0706** [comment]
Cool! How does one use this?

`-d "line1" "line2" "line3"`

Like that?

**Comment by tedchamb** [comment]
Yes, that is correct.   We should have this version released soon.
